hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/FileStore.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.service.file.impl;
+
+import com.hedera.hapi.node.base.Timestamp;
+import com.hedera.hapi.node.state.file.File;
+import com.hedera.node.app.service.file.FileMetadata;
+
+import java.util.Optional;
+
+/**
+ * Base class for {@link ReadableFileStoreImpl} and {@link WritableFileStoreImpl}.
+ */
+public class FileStore {
+    public static FileMetadata fileMetaFrom(final File file) {
```

Comment:
@hendrikebbers I can add `@Nonnull` but I'm using it in `ReadableFileStoreImpl` and it use orElse that just if all fields exist it will fulfill otherwise it will be null

Reply:
done